src/visualizers/widgets/ExecutionIndex/ExecutionIndexWidget.js

```diff
@@ -64,6 +65,14 @@ define([
         this.logger.warn('No node id found for node-nav!');
     };
 
+    ExecutionIndexWidget.prototype.onExecutionDelete = function (event) {
+        let target = event.target,
+            id = target.getAttribute('data-id');
+
+        if(id)
```

Comment:
Add curly braces. This would also be good to add a rule about this to code climate...